src/main/java/org/nekobehemoth/hw06/server/controller/MyController.java

```diff
@@ -0,0 +1,74 @@
+package org.nekobehemoth.hw06.server.controller;
+
+import lombok.Getter;
+import org.nekobehemoth.hw06.server.ServerStatistics;
+import org.nekobehemoth.hw06.server.annotations.Controller;
+import org.nekobehemoth.hw06.server.annotations.GetMapping;
+import org.nekobehemoth.hw06.server.annotations.PostMapping;
+import org.nekobehemoth.hw06.server.http.HttpRequest;
+import org.nekobehemoth.hw06.server.http.HttpResponse;
+import org.nekobehemoth.hw06.server.httpmethodhandlers.GetMethodHandler;
+import org.nekobehemoth.hw06.server.httpmethodhandlers.HttpMethodHandler;
+import org.nekobehemoth.hw06.server.utils.Utils;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+@Controller
+public class MyController {
+    @Getter
+    public final Map<String, HttpMethodHandler> methodHandlerMap = new HashMap<>();
+
+    ServerStatistics serverStatistics;
+
+    public MyController() {
+
+    }
+
+    public MyController(ServerStatistics serverStatistics) {
+        this.serverStatistics = serverStatistics;
+    }
+
+
+    @GetMapping("/")
+    public void getHomePage(HttpRequest httpRequest, HttpResponse httpResponse) throws IOException {
```

Comment:
for a fully functional hw07 we also need parsing input parameters based on method arguments types
not exactly hw07 yet, but close ! good job!